### DISCUSSION TITLE
"Package ctex is too old"问题

### DISCUSSION THREAD

**Discussion by MusicJIA** [stone-zeng/fduthesis#271] [Q&A]
要求“ctex” 版本大于 v2.4.10，查看ctex宏包信息，ctex宏包版本确实大于2.4.10 （实际版本为：2.5.8）。
但仍然报错：
Package fduthesis Error: Package "ctex" is too old.

**Comment by stone-zeng** [comment]
可以传一下编译的 log 文件吗？感觉可能是调用了发行版自带的模板文件。

**Reply by MusicJIA** [reply]
[log.docx](https://github.com/stone-zeng/fduthesis/files/9484795/log.docx)

**Reply by stone-zeng** [reply]
可以发现里面有一行：

```
(/usr/local/texlive/2022/texmf-dist/tex/latex/fduthesis/fduthesis.cls
```

所以确实是调用了发行版自带的模板，而没有用新的。可以参考文档生成一下，或者更新 TeX Live（今天刚上传，可能要到明天才能更新到）。

**Reply by MusicJIA** [reply]
非常感谢您的耐心解答！

**Reply by MusicJIA** [reply]
您提到的 “（今天刚上传，可能要到明天才能更新到）” ，意思是等更新到以后，我直接下载更新后的版本就不会出现问题了，对吗？

**Reply by note286** [reply]
@MusicJIA TeX Live使用CTAN的镜像，镜像同步会在24小时内完成，一般会有延时，你可以稍后更新即可。

**Reply by MusicJIA** [reply]
@note286 谢谢您的解答！
